deployment/appserver/docker-compose.yml

```diff
@@ -53,11 +53,13 @@ services:
       - SB_ANALYTICS_ID
       - SB_DATADOG_KEY
       - SB_DISABLE_CRAWLERS
+      - GOOGLE_APPLICATION_CREDENTIALS=/google-credentials.json
```

Comment:
What you wanted was "a file named `google-credentials.json` next to this file", what you wrote was "a file named `google-credentials.json` at the root of the filesystem"

Reply:
Oh nevermind, I forgot this is going to be where it is inside the container, this is fine